rest/server_context.go

```diff
@@ -72,7 +72,7 @@ type bootstrapContext struct {
 func (sc *ServerContext) CreateLocalDatabase(dbs DbConfigMap) error {
 	for _, dbConfig := range dbs {
 		dbc := dbConfig.ToDatabaseConfig()
-		_, err := sc._getOrAddDatabaseFromConfig(*dbc, false, false)
+		_, err := sc._getOrAddDatabaseFromConfig(*dbc, false, db.ConnectToBucket)
```

Comment:
Might be nicer to continue using the `db.GetConnectToBucketFn` helper but just with a static `false` parameter - fewer places to change if `ConnectToBucket` needs to change/move.